portal-ui/src/screens/Console/Heal/Heal.tsx

```diff
@@ -0,0 +1,343 @@
+import { HorizontalBar } from "react-chartjs-2";
+import React, { useEffect, useState } from "react";
+import {
+  Button,
+  Grid,
+  Typography,
+  TextField,
+  Checkbox,
+} from "@material-ui/core";
+import { IMessageEvent, w3cwebsocket as W3CWebSocket } from "websocket";
+import { createStyles, Theme, withStyles } from "@material-ui/core/styles";
+import { wsProtocol } from "../../../utils/wsUtils";
+import api from "../../../common/api";
+import { FormControl, MenuItem, Select } from "@material-ui/core";
+import { BucketList, Bucket } from "../Watch/types";
+import { HealStatus } from "./types";
+import { niceBytes } from "../../../common/utils";
+
+const styles = (theme: Theme) =>
+  createStyles({
+    watchList: {
+      background: "white",
+      maxHeight: "400px",
```

Comment:
Please use only numeric values in createStyles function when using pixels
```suggestion
      maxHeight: 400,
```

Reply:
done